x-pack/plugins/alerts/server/task_runner/task_runner.test.ts

```diff
@@ -184,11 +184,15 @@ describe('Task Runner', () => {
     expect(eventLogger.logEvent).toHaveBeenCalledTimes(1);
```

Comment:
the changes at the top of this file are a dog's breakfast, sorry.  The order the events get logged has changed, where the `execute` event is written AFTER all the other events now, but post-dated to the start of the execution process (and will be <= the other events generated), since we now have to write it AFTER all the execution processing.  So, the diff is not very pleasing ...